app/forms/security_event_form.rb

```diff
@@ -116,23 +119,23 @@ def validate_jwt
 
     if error_code && error_message
       @error_code = error_code
-      errors.add(:jwt, error_message)
+      errors.add(:jwt, error_message, type: :"#{error_message.split('.').last}")
```

Comment:
maybe it would be better to set a `error_reason` than to try to infer from the translated error emssage?